integration-test/run.sh

```diff
@@ -51,8 +51,9 @@ SKELETON=$(cd $HERE/skeleton; echo $PWD)
 : "${RESOLVER_SERIES:=lts-13}"
 
 # Find the latest resolver in the series to use.
-RESOLVER=$(curl -s https://www.stackage.org/download/snapshots.json | \
-               jq -r '."'$RESOLVER_SERIES'"')
+# RESOLVER=$(curl -s https://www.stackage.org/download/snapshots.json | \
```

Comment:
Can this commented code be removed?

Reply:
Done in https://github.com/seek-oss/serverless-haskell/pull/101/commits/bd18b7b648ff65cdfd713a615f146ceafad89ece thanks @axman6